### PR TITLE
Praetorian Acid Spray Now Goes Past Razorwire

### PR DESCRIPTION
**Pull request by Surrealaser**
:cl: by Surrealistik
fix: Acid Spray now goes past razorwire.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/xenomorph/Powers.dm [hunk 1]
```diff
@@ -276,7 +282,11 @@
 							var/obj/vehicle/multitile/root/cm_armored/A = O
 							A.take_damage_type(rand(40,60) + 8 * upgrade, "acid", src)
 							A.healthcheck()
-						normal_density_flag = 1
+						else if(istype(O, /obj/structure/razorwire) )
```
Comment (DominikPanic) [inline code/modules/mob/living/carbon/xenomorph/Powers.dm]: Another snowflake checks that should be handled by components.

Comment (Surrealaser) [reply code/modules/mob/living/carbon/xenomorph/Powers.dm]: Well this is currently the norm unfortunately.

Comment (Surrealaser) [reply code/modules/mob/living/carbon/xenomorph/Powers.dm]: If this is cause for barring the PR, we should just delete the entire ability.

Comment (spookydonut) [reply code/modules/mob/living/carbon/xenomorph/Powers.dm]: We dont have components yet so this is fine

**Pull request merged**